stdlib/public/core/FloatingPointParsing.swift.gyb

```diff
@@ -120,16 +145,16 @@ extension ${Self}: LosslessStringConvertible {
   ///       // p?.isNaN == true
   ///       // String(p!) == "nan(0x10)"
   ///
-  /// Passing any other format or any additional characters as `text` results
-  /// in `nil`. For example, the following conversions result in `nil`:
+  /// - A value in any other format or containing additional characters
+  ///   results in a `nil` value. For example, the following conversions
+  ///   result in `nil`:
   ///
-  ///     ${Self}(" 5.0")      // Includes whitespace
-  ///     ${Self}("±2.0")      // Invalid character
-  ///     ${Self}("0x1.25e4")  // Incorrect exponent format
+  ///       ${Self}(" 5.0")      // Includes whitespace
+  ///       ${Self}("±2.0")      // Invalid character
+  ///       ${Self}("0x1.25e4")  // Incorrect exponent format
   ///
-  /// - Parameter text: The input string to convert to a `${Self}` instance. If
-  ///   `text` has invalid characters or is in an invalid format, the result
-  ///   is `nil`.
+  /// - Parameter text: The input string to convert to a `${Self}?` instance.
+  ///   The value will be `nil` if the input is incorrectly formatted.
```

Comment:
What issue were you fixing by rewriting this bit?  "If the input is incorrectly formatted, the value is `nil`." is a better fit for the reference style guidelines.  Because this fact is about the init as a whole, not just about the parameter, my inclination would be to move it to the start of the discussion, but I can see an argument for calling it out because here because it's the parameter's value that triggers this behavior.

Reply:
The entirety of the documentation is about how this one parameter is interpreted.  I'm inclined to just delete the reference to `nil` entirely, since that just duplicates the content immediately above.